crates/matrix-sdk/CHANGELOG.md

```diff
@@ -108,6 +108,9 @@ causing deleted room avatars to not be flagged is fixed. ([#5293](https://github
   ([#5047](https://github.com/matrix-org/matrix-rust-sdk/pull/5047))
 - `Room::set_unread_flag()` is now a no-op if the unread flag already has the wanted value.
   ([#5055](https://github.com/matrix-org/matrix-rust-sdk/pull/5055))
+- Support for the deprecated `GET /auth_issuer` endpoint was removed in the `OAuth` API. Only the
+  `GET /auth_metadata` endpoint is used now.
+  ([#5302](https://github.com/matrix-org/matrix-rust-sdk/pull/5302))
```

Comment:
We started to add new entries to the top, since a couple of entries ended up in the wrong place due to a release happening between the opening of a PR and the merge of the PR.

Reply:
Done.